src/rules/at-rule-blockless-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,42 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("always", tr => {
+  tr.ok("@import 'x.css';\na {}")
+  tr.ok("@charset 'UTF-8';\na {}")
+  tr.ok("@namespace url(XML-namespace-URL);\na {}")
+  tr.ok("@import 'x.css'); /* comment */\n")
```

Comment:
If `@import 'x.css';` is the only or last line in the document, I think this rule probably should ignore it. Agree? If so, let's add that test.